src/shared/test/frame-error-capture-test.js

```diff
@@ -0,0 +1,144 @@
+import { delay } from '../../test-util/wait';
+import {
+  captureErrors,
+  handleErrorsInFrames,
+  sendError,
+  sendErrorsTo,
+} from '../frame-error-capture';
+
+describe('shared/frame-error-capture', () => {
+  let errorEvents;
+
+  function handleMessage(event) {
+    if (event.data.type === 'hypothesis-error') {
+      errorEvents.push(event.data);
+    }
+  }
+
+  beforeEach(() => {
+    errorEvents = [];
+    window.addEventListener('message', handleMessage);
+  });
+
+  afterEach(() => {
+    window.removeEventListener('message', handleMessage);
+    sendErrorsTo(null);
+  });
+
+  describe('captureErrors', () => {
```

Comment:
I agree. I did this originally because `captureErrors` was the "main" API that other modules used. I've split this now so that `captureErrors` has a basic test that an error gets sent in the happy path, and there are now explicit tests for `sendErrors` that handle the failure cases.